attachment_image_resize/models/ir_attachment.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2024 Quartile
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+IMAGE_TYPES = ["image/png", "image/jpeg", "image/bmp", "image/tiff"]
+
+
+class IrAttachment(models.Model):
+    _inherit = "ir.attachment"
+
+    resize_done = fields.Boolean()
+
+    def _postprocess_contents(self, values):
+        self = self.with_context(model=values.get("res_model"))
```

Comment:
`model` is too generic and may invite conflicting situations.
```suggestion
        self = self.with_context(resize_target_model=values.get("res_model"))
```